sdk/js/src/utils/consts.ts

```diff
@@ -308,6 +310,11 @@ const MAINNET = {
     token_bridge: undefined,
     nft_bridge: undefined,
   },
+  neutron: {
+    core: undefined,
```

Comment:
The wormhole-ibc contract is deployed on Neutron mainnet with address neutron16rerygcpahqcxx5t8vjla46ym8ccn7xz7rtc6ju5ujcd36cmc7zs9zrunh